### PR TITLE
Add Session tests

### PR DESCRIPTION
**Pull request by JonahhGohh**
#77 #61

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AddSessionCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    public void equals() {
+        Session session = new SessionBuilder().build();
+        Session anotherSession = new SessionBuilder().withSessionDate("2020-12-12", "10:00").build();
+        AddSessionCommand addSessionCommand = new AddSessionCommand(session, ALICE.getName());
+        AddSessionCommand anotherAddSessionCommand = new AddSessionCommand(anotherSession, BOB.getName());
+
+        assertTrue(addSessionCommand.equals(addSessionCommand));
+        assertTrue(anotherAddSessionCommand.equals(anotherAddSessionCommand));
+
+        assertFalse(addSessionCommand.equals(1));
```
Comment (enhao25) [inline src/test/java/seedu/address/logic/commands/AddSessionCommandTest.java:72]: Looks good! Just wondering if there could be comments here similar to ModelManagerTest.java equals method, to better allow the others reading this code to understand the test cases here better.

**Review by enhao25 — APPROVED** [review]
Yeap LGTM. Just 1 small comments on adding comments to the equals method. Previously in the group, I think shion also mention that changing the session to store student instead of having student store session could be a possible idea. If we somehow decide to do that, not sure if some of the test case needs to be changed. However, at this point, it might not be time efficient to change, might need to discuss this future when we meet tmr, to see if there is a point in doing that. Else, looks good

File: src/test/java/seedu/address/logic/commands/AddSessionCommandTest.java (hunk 1 above)
Comment (JonahhGohh) [reply src/test/java/seedu/address/logic/commands/AddSessionCommandTest.java:72]: Ok, thanks for the suggestion. I will add some comments to the test cases to make it clearer. 👍

**Comment by JonahhGohh** [comment]
> Yeap LGTM. Just 1 small comments on adding comments to the equals method. Previously in the group, I think shion also mention that changing the session to store student instead of having student store session could be a possible idea. If we somehow decide to do that, not sure if some of the test case needs to be changed. However, at this point, it might not be time efficient to change, might need to discuss this future when we meet tmr, to see if there is a point in doing that. Else, looks good

I think regarding this, wouldn't it be better to just have these test cases now and modify them later if we were to introduce the functionality?

**Pull request merged**